README.md

```diff
@@ -2,78 +2,16 @@
 SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 -->
 
-# beman.example: A Beman Library Example
+# beman.dump: Implementation of proposed `std::dump`
 
-![Continuous Integration Tests](https://github.com/beman-project/Example/actions/workflows/ci_tests.yml/badge.svg)
+![Continuous Integration Tests](https://github.com/beman-project/dump/actions/workflows/ci_tests.yml/badge.svg)
 
-`beman.example` is a minimal C++ library conforming to [The Beman Standard](https://github.com/beman-project/beman/blob/main/docs/beman-standard.md). This can be used as a template for those intending to write Beman libraries. It may also find use as a minimal and modern  C++ project structure.
+`std::dump` prints its arguments space-separated with a new-line.
```

Comment:
```suggestion
`beman::dump::dump()` prints its arguments space-separated with a new-line. This
is useful as a debugging utility.
```

It's important to explain to users when an why they would use this library; what it's purpose is.